lib/gitlab_custom_hook.rb

```diff
@@ -48,11 +48,14 @@ def call_receive_hook(hook, changes)
       # need to close stdin before reading stdout
       stdin.close
 
+      stdout_stderr.each_line do |line|
+        puts line
+        $stdout.flush
```

Comment:
You can set `$stdout.sync = true` to automatically flush everything that's written to STDOUT rather than do it manually.